### DISCUSSION TITLE
Should there be an issue to track publishment of this?

### DISCUSSION THREAD

**Discussion by beedell-roke** [trakt-tools/universal-trakt-scrobbler#213] [Q&A] [answered]
Am I allowed to create an issue to track the release of this to web stores? I only want to be notified of that, so tracking the releases etcetera is unnecessary for me.

**Comment by MrMamen** [comment, marked as answer]
It should be released when this issue is resolved:
https://github.com/trakt-tools/universal-trakt-scrobbler/issues/147
